backend/cmd/headlamp.go

```diff
@@ -209,14 +210,27 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 
 	// KubeConfig clusters
 	if kubeConfigPath != "" {
-		var err error
-
-		contextsFound, err := GetContextsFromKubeConfigFile(kubeConfigPath)
-		if err != nil {
-			log.Println("Failed to get contexts from", kubeConfigPath, err)
+		var kubeConfigs []string
+		if runtime.GOOS == "windows" {
+			kubeConfigs = strings.Split(kubeConfigPath, ";")
+		} else {
+			kubeConfigs = strings.Split(kubeConfigPath, ":")
 		}
 
-		contexts = append(contexts, contextsFound...)
+		for _, kubeConfig := range kubeConfigs {
+			kubeConfig, err := absPath(kubeConfig)
+			if err != nil {
+				log.Println("Failed to get current user", err)
+				continue
+			}
+
+			contextsFound, err := GetContextsFromKubeConfigFile(kubeConfig)
+			if err != nil {
+				log.Println("Failed to get contexts from", kubeConfig, err)
```

Comment:
Should we print these errors to stderr?

Reply:
All the logs are in Println, Ill fix it as a part of another PR ?